packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -172,8 +172,12 @@ class Auth {
     private silenceWarnings: boolean
 
     constructor(config: AuthConfig) {
+        // Special endpoint for injecting SLAS private client secret
+        const baseUrl = config.proxy.split(`/mobify/proxy/api`)[0]
+        const privateClientEndpoint = `${baseUrl}/ssr/auth`
+
         this.client = new ShopperLogin({
-            proxy: config.proxy,
+            proxy: config.clientSecret ? privateClientEndpoint : config.proxy,
```

Comment:
Only update the proxy for ShopperLogin because the ShopperCustomer client in Auth is used only for new customer registration and does not require the SLAS client secret.

This does mean all ShopperLogin requests will route to the custom proxy so the custom proxy has extra logic for specifically injecting the secret just for the SLAS getToken endpoint.